digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/csvDataImporter/TrafficLightsCsvImporter.scala

```diff
@@ -19,15 +19,28 @@ class TrafficLightsCsvImporter(roadLinkServiceImpl: RoadLinkService, eventBusImp
 
   lazy val trafficLightsService: TrafficLightService = new TrafficLightService(roadLinkService)
 
+  override def verifyData(parsedRow: ParsedProperties, user: User): ParsedCsv = {
+    val optLon = getPropertyValueOption(parsedRow, "lon").asInstanceOf[Option[BigDecimal]]
+    val optLat = getPropertyValueOption(parsedRow, "lat").asInstanceOf[Option[BigDecimal]]
+
+    (optLon, optLat) match {
+      case (Some(lon), Some(lat)) =>
+        val roadLinks = roadLinkService.getClosestRoadlinkForCarTrafficFromVVH(user, Point(lon.toLong, lat.toLong), false)
+        roadLinks.isEmpty match {
+          case true => (List(s"No Rights for Municipality or nonexistent road links near asset position"), Seq())
+          case false => (List(), Seq(CsvAssetRowAndRoadLink(parsedRow, roadLinks)))
```

Comment:
Avoid the match with true and false.
Even intellij suggest to change it to an IF

Reply:
Done.